docs/patterns/index.md

```diff
@@ -24,6 +24,12 @@ These design patterns are useful for building reliable, scalable, secure applica
 
 Each pattern describes the problem that the pattern addresses, considerations for applying the pattern, and an example based on Microsoft Azure. Most patterns include code samples or snippets that show how to implement the pattern on Azure. However, most patterns are relevant to any distributed system, whether hosted on Azure or other cloud platforms.
 
+> [!NOTE]  
+> In distributed systems:
+> - Versioning isn't simple.
+> - Observabality is not optional but mandatory.
+
+
```

Comment:
Convert this text to be inline instead of using the NOTE syntax. Find a way to stress the importance of these topics and relate it to the list below (presumably these will help solve those problems)